boot/zephyr/Kconfig

```diff
@@ -14,36 +14,21 @@ config MCUBOOT
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 	select USE_CODE_PARTITION if HAS_FLASH_LOAD_OFFSET
 
-# Define used by partition_manager.py to deduce size of partition
-config PM_PARTITION_SIZE_MCUBOOT
-	hex "Flash space reserved for bootloader."
-	default 0xc000
-	help
-	  Flash space set aside for the MCUBoot. Note, the name
-	  of this configuration needs to match the requirements set by the
-	  script 'partition_manager.py'. See pm.yaml.
+partition=MCUBOOT
```

Comment:
add comment: Note, the name of this configuration needs to match the requirements set by the script 'partition_manager.py'. See pm.yaml."

Reply:
Fixed